proto/cosmos/upgrade/v1beta1/upgrade.proto

```diff
@@ -54,13 +54,13 @@ message SoftwareUpgradeProposal {
   option (gogoproto.equal)                   = true;
 
   // title of the proposal
-  string title       = 1;
+  string title = 1;
```

Comment:
Same question, it is done by `make proto-format` or not?

Reply:
Correct, from `make proto-all`